src/makeInfinityAdder.js

```diff
@@ -4,7 +4,23 @@
  * @return {function}
  */
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  const call = (...args) => {
+    if (args.length === 0) {
+      const result = sum;
+
+      sum = 0;
+
+      return result;
+    } else {
+      sum += args[0];
+
+      return call;
```

Comment:
Consider using the rest parameter syntax `...args` only if you expect to handle multiple arguments. Since the adder function is intended to be called with either one argument or none, you can simplify the parameter handling by using a single parameter instead.

Reply:
[comment](https://github.com/mate-academy/js_task-inifinite-sum-closures/pull/2028/files#r1683618128) not fixed